Fotoblog.DAL/Entities/PhotoEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Fotoblog.DAL.Entities
+{
+    public class PhotoEntity
+    {
+        public int Id { get; set; }
+        public string Title { get; set; } = string.Empty;
+        public string ?Description { get; set; }
+        public string ImagePath { get; set; } = string.Empty;
+        public DateTime CreateDate { get; set; }
+
+        public ICollection<TagEntity> ?Tags { get; set; }
```

Comment:
Użyj List zamiast ICollection. Nullable też niepotrzebne. Jeśli dobrze pamiętam kolekcje są nullable z defaultu (mogę się mylić).

Reply:
bez ? lista to też non nullable property, ale można przypisać od razu pustą listę